src/components/Header/index.tsx

```diff
@@ -17,8 +17,8 @@ const Header = (): JSX.Element => {
     if (isKeyPress) {
       return;
     }
-    const target = e.target as HTMLElement;
-    const toggle = target.innerHTML.includes('nights_stay') ? 'dark' : 'light';
+    const target = e.target as any;
```

Comment:
Please figure out a way to do this without using `any`

Reply:
I tried to do it using HTMLElement interface, which should have the innerText property but my IDE threw an error when I tried to access it. Something else I did was the theme changing only when the button was pressed, so the interfaces remained the same, I just added an extra condition to the if statement. What do you think I should do?

Reply:
The build is failing in the case of `any`, and not in the case of `HTMLElement`. Can you check your IDE settings once as it seems like it was throwing a wrong error?

Reply:
Yes I also tried with HTMLElement type and its not throwing me any kind of error. It works on my Machine 😄.

Reply:
Reinstalling my modules seems to have fixed it, I've pushed a new commit.